bottlerocket/agents/src/userdata.rs

```diff
@@ -0,0 +1,158 @@
+// Heavily borrowed from Bottlerocket's merge-toml crate.
+// See https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/storewolf/merge-toml/src/lib.rs
```

Comment:
quick nit: sorry, this should be a more permanent link that's tagged:
```suggestion
// See https://github.com/bottlerocket-os/bottlerocket/blob/v1.11.1/sources/api/storewolf/merge-toml/src/lib.rs
```